File: BRaVeMobile/app/src/main/java/intl/iom/bravemobile/models/DataPacketHeader.java
package intl.iom.bravemobile.models;

public class DataPacketHeader {
    public String EncryptedKey ;
    public String Nonce ;
    public String ActivityCode ;
    public String DPoP;
    public String Timestamp ;
    public String Extra = "" ;
    public String BatchId;
}

File: BRaVe-Mobile-Backend/Models/DataPacketHeader.cs
using System.Text.Json.Serialization;

namespace BRaVe_Mobile_Backend.Models
{
    public class DataPacketHeader
    {
        //[JsonPropertyName("encryptedKey")]
        public string EncryptedKey { get; set; }

        //[JsonPropertyName("nonce")]
        public string Nonce { get; set; } //Batch Id

        //[JsonPropertyName("activityCode")]
        public string ActivityCode { get; set; }

        //[JsonPropertyName("dPoP")]
        public string DPoP { get; set; }
        public string BatchId { get; set; }

        //[JsonPropertyName("timestamp")]
        public string Timestamp { get; set; }

        public string? Extra { get; set; }

    }
}
